### ISSUE TITLE
openedClass not removed when close/dismiss with 2 pop-ups

### ISSUE THREAD

**Issue by povilass** [angular-ui/bootstrap#4184]
I have problem, one case like you have 2 opened pop-ups and i got 2 body classes ' class="modal-scrollable modal-open" ' when I close pop-up with modal-open class, class is not toggled 
```javascript
body.toggleClass(modalWindow.openedClass || OPENED_MODAL_CLASS, openedWindows.length() > 0).
```

**Comment by wesleycho** [comment]
For reference, some conversation is in #4171.

**Comment by povilass** [comment]
Even if you have 3 pop-ups they should use 3 different classes which should be removed not toggled.

**Comment by povilass** [comment]
```javascript
if(openedWindows.length() == 0) {
	body.removeClass(modalWindow.openedClass || OPENED_MODAL_CLASS);
}
```

**Comment by wesleycho** [comment]
It is more complicated than that - a separate data structure is likely necessary to store the class list, since there is the possibility that someone used the same custom class in multiple modals.

**Comment by povilass** [comment]
Ye only way it can work with storing those classes and counting them maybe.

**Comment by povilass** [comment]
So any progress ?

**Comment by wesleycho** [comment]
Not yet, but I did not forget about this issue - I'm a busy person, but I will make sure to get a fix before we release 0.13.4, so don't worry unless I break that promise :) .

**Comment by povilass** [comment]
Like for you, waiting for new version!!
